notebooks/tutorial/2 - Download data.ipynb

```diff
@@ -89,23 +69,36 @@
    "outputs": [],
    "source": [
     "import os\n",
-    "from pureskillgg_dsdk.exchange import download_dataexchange_dataset_revision"
+    "import requests\n",
```

Comment:
Interesting, when I looked up how to do this I was suggested to use `import urllib` and `urllib.request`. Is `requests` also in std lib? Two ways to do the same thing in Python? 😲

Reply:
my google search led me to this and i didn't look more into it haha https://www.codingem.com/python-download-file-from-url/

Requests is mentioned in the top yellow box here https://docs.python.org/3/library/urllib.request.html#module-urllib.request

Reply:
Yea, apparently I used the legacy interface: https://docs.python.org/3/library/urllib.request.html#urllib.request.urlretrieve at https://github.com/pureskillgg/dsdk/blob/6f7dcc064e6a4778fc268e2eb0c191fbd5f6d965/pureskillgg_dsdk/exchange/dataset.py#L75

Anyway, we will need to add this with poetry yea?

Reply:
😮  yup. done.